### DISCUSSION TITLE
part-ii/2019-26/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#162] [Question comments]
Part II 2019 Automata and Formal Languages: Paper 1, Section II, H 

https://questions.tripos.org/part-ii/2019-26/

**Comment by artie2000** [comment]
Hint: If you have a binary word $w$ with numerical value $n_w$ and append a binary digit $\sigma\in\{0,1\}$, then

$$n_{w\sigma}=2n_w+n_\sigma$$.
